Tools/cases_generator/analyzer.py

```diff
@@ -667,8 +684,8 @@ def find_stmt_end(node: parser.InstDef, idx: int) -> lexer.Token:
         if tkn.kind == "SEMI":
             return node.block.tokens[idx+1]
 
-def check_escaping_calls(instr: parser.InstDef, escapes: dict[lexer.Token, tuple[lexer.Token, lexer.Token]]) -> None:
-    calls = {escapes[t][0] for t in escapes}
+def check_escaping_calls(instr: parser.InstDef, escapes: dict[lexer.Token, EscapingCall]) -> None:
+    calls = {escapes[t].call for t in escapes}
```

Comment:
```suggestion
    calls = {t.call for t in escapes.values()}
```

saves a dict lookup in the expr.